findbugs/build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
 }
 
 apply from: "$rootDir/gradle/jacoco.gradle"
+apply from: "$rootDir/gradle/maven.gradle"
```

Comment:
I put this `apply` to not root `build.gradle` but subprojects, to stop deploying test case projects and Eclipse plugin.